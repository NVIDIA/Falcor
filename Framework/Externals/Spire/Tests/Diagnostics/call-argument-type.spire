// call function with wrong argument type

struct A {};
struct B {};

void f(A a) {}
void g(B b)
{
	f(b);
}
