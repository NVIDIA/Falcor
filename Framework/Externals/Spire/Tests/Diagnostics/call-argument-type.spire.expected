result code = -1
standard error = {
Tests/Diagnostics/call-argument-type.spire(9): error 30019: expected an expression of type 'A', got 'B'
}
standard output = {
}
